### ISSUE TITLE
Include a dist version

### ISSUE THREAD

**Issue by mdesantis** [atomiks/tippyjs#17]
Hello,

I'm a server-side old-fashioned web developer and I'm a newbie to Javascript compiling, bundling and stuff like that. I noticed that Javascript proejcts usually provide a dist directory with the compiled version of the library they develop, could this project do it too? It would help people like me who don't have Webpack / don't know how to configure it / tried and failed.

EDIT: I noticed there is a production version, but it's not provided when I run `yarn install tippy.js` (yarn is actually the Javascript dependency manager I'm using at the moment), could them be inlcuded in the repository?

Thank you for your attention and for the library too :) sharing is caring!

**Comment by atomiks** [comment]
Check releases, latest is here: https://github.com/atomiks/tippyjs/releases/tag/v0.2.8

This includes the .js and .css files you'll need.

**Comment by mdesantis** [comment]
Sorry for the late response. The production files are not available when you install tippy with `yarn install tippy.js`, I just would like to manage my dependencies using yarn, instead of download a zip and check from time to time if the library has updates.

**Comment by mdesantis** [comment]
For example, now I have to download the 0.3.0 in order to update the 0.2.8, and I know that there is a 0.3.0 version just beacuse I noticed that it's written in the url you wrote.

**Comment by atomiks** [comment]
Tippy is using npm instead of yarn

**Comment by mdesantis** [comment]
tippy uses yarn (or yarn uses tippy, I don't know how it works): https://yarnpkg.com/en/package/tippy.js

Anyway it seems not to be different, if I run `npm install tippy.js` I get files that I should compile somehow

**Comment by atomiks** [comment]
I had no idea it was even on there... haven't used yarn before so maybe it automatically gets put on there from npm or github, not sure

Also you'd still need the CSS file updates which aren't on there, I think it's only the JS?

**Comment by mdesantis** [comment]
I guess your project is published in almost all nodejs package managers without your knowledge :D

Yeah I would need also the .css file too, like for instance in this project I use: 

https://github.com/madebymany/sir-trevor-js

If you notice, in the repoository there isn't any `build` folder, but when you run `npm install --save sir-trevor` I get

```
> ls node_modules/sir-trevor/build/
sir-trevor.css  sir-trevor-icons.svg  sir-trevor.js  sir-trevor.min.css  sir-trevor.min.js
```

which are the assets I need inside my web app. Frankly I don't know how it works, I guess it's something related to `npm publish`.

> Anyway I think v0.3.3 should be the last update for a while by myself until issues are pointed out by others 

Yeah I can live without `yarn add tippy.js`/`npm install tippy.js`, but it would be more convenient. This is the only dependency I have to hard-code in my repository :D but take your time don't worry

**Comment by atomiks** [comment]
Alright... I published a new version (v0.3.4, can't overwrite old one) with a `dist` folder now (and removed unncessary docs folder). There's also a `src` if you need the unminified too.

**Comment by mdesantis** [comment]
Perfect! I'm so happy now :D thank you!!!
